src/ProductView/TabBar/QueryInterface/QueryInterface.js

```diff
@@ -83,7 +86,8 @@ class QueryInterface extends Component {
     }
 
     handleChangeEventTypeName(name) {
-        this.setState({ eventTypeName: name });
+        const modalIsAbleToSave = this.isModalAbleToSave(name, this.state.eventQuery);
```

Comment:
Renaming isModalAbleToSave -> queryIsSafeable makes sense here. Our main target is the eventType (and its query of course). Modal ist only an UI helper which should not be reflected in namings.